packages/components/psammead-brand/src/index.jsx

```diff
@@ -105,4 +107,39 @@ Brand.propTypes = {
   }).isRequired,
 };
 
+StyledBrand.propTypes = brandProps;
+
+const Brand = ({ brandName, svgHeight, minWidth, maxWidth, svg, url }) => (
+  <Banner svgHeight={svgHeight}>
+    {url ? (
+      <StyledLink href={url} maxWidth={maxWidth} minWidth={minWidth}>
+        <StyledBrand
+          brandName={brandName}
+          svg={svg}
+          maxWidth={maxWidth}
+          minWidth={minWidth}
+          url={url}
```

Comment:
I dont think brand needs url

Reply:
or min width by the looks of it, can you review this file for needed prop types please